exiftool.go

```diff
@@ -377,3 +413,18 @@ func SetExiftoolBinaryPath(p string) func(*Exiftool) error {
 		return nil
 	}
 }
+
+// Debug enables printing debug logs to stderr
```

Comment:
The comment does not fit with the code: any `io.Reader` can be used and `os.Stderr` is the default one (if none is provided).

Reply:
Ya, I wrote that before refactoring for testing, thanks, I'll fix it.

Reply:
fixed